### PR TITLE
Produce/publish the VS.Redist.Common.Net.Core.SDK.MSBuildExtensions artifacts

### PR DESCRIPTION
**Pull request by ghost**
Produce/publish the "VS.Redist.Common.Net.Core.SDK.MSBuildExtensions" artifacts; drop to "CLI" not "SDK"

Put back the infrastructure to generate the 'VS.Redist.Common.Net.Core.SDK.MSBuildExtensions' artifacts

### CODE REVIEW

File: packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.x64.nuspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dsplaisted) [inline packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.x64.nuspec]: I don't think this nuspec is required for generating the MSBuildExtensions insertion package.

File: packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.x86.nuspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dsplaisted) [inline packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.x86.nuspec]: I don't think this .nuspec is required either.

File: packaging/windows/clisdk/bundle.thm [hunk 3]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dsplaisted) [inline packaging/windows/clisdk/bundle.thm]: I think these bundle and dotnet files are for the .NET Core SDK MSI, and shouldn't be needed in this repo.

File: packaging/windows/clisdk/generatebundle.ps1 [hunk 4]
```diff
@@ -0,0 +1,130 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
```
Comment (dsplaisted) [inline packaging/windows/clisdk/generatebundle.ps1]: I don't think this script is used to generate the MSBuildExtensions package.

File: packaging/windows/clisdk/generatemsi.ps1 [hunk 5]
```diff
@@ -0,0 +1,140 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
```
Comment (dsplaisted) [inline packaging/windows/clisdk/generatemsi.ps1]: I don't think this script is needed in this repo.

File: packaging/windows/clisdk/provider.wxs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```
Comment (dsplaisted) [inline packaging/windows/clisdk/provider.wxs]: I think this file isn't needed.

File: packaging/windows/clisdk/registrykeys.wxs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```
Comment (dsplaisted) [inline packaging/windows/clisdk/registrykeys.wxs]: I think this file isn't needed.

File: packaging/windows/clisdk/variables.wxi [hunk 8]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```
Comment (dsplaisted) [inline packaging/windows/clisdk/variables.wxi]: I think this file isn't needed.

File: packaging/windows/msbuildextensions/generatemsi.ps1 [hunk 9]
```diff
@@ -0,0 +1,140 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
```
Comment (dsplaisted) [inline packaging/windows/msbuildextensions/generatemsi.ps1]: I think this file isn't needed.

File: packaging/windows/msbuildextensions/msbuildextensions.wxs [hunk 10]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dsplaisted) [inline packaging/windows/msbuildextensions/msbuildextensions.wxs]: I think this file isn't needed.

File: packaging/windows/msbuildextensions/provider.wxs [hunk 11]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```
Comment (dsplaisted) [inline packaging/windows/msbuildextensions/provider.wxs]: I think this file isn't needed.

File: packaging/windows/msbuildextensions/registrykeys.wxs [hunk 12]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```
Comment (dsplaisted) [inline packaging/windows/msbuildextensions/registrykeys.wxs]: I think this file isn't needed.

File: packaging/windows/msbuildextensions/variables.wxi [hunk 13]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```
Comment (dsplaisted) [inline packaging/windows/msbuildextensions/variables.wxi]: I think this file isn't needed.

**Review by dsplaisted — COMMENTED** [review]
We shouldn't need to generate an MSI for MSBuildExtensions anymore, so you can delete a whole bunch of the files that support that from this PR.

Also, please fix the spelling of `MSBuildExtentions` to `MSBuildExtensions`

**Comment by johnbeisner** [comment]
@dotnet-bot help

**Comment by johnbeisner** [comment]
@dotnet-bot test this please

**Comment by johnbeisner** [comment]
@dsplaisted 
Can I get an approval please.

File: build/Package.targets [hunk 14]
```diff
@@ -2,21 +2,23 @@
...
   <Target Name="GenerateInstallers"
-          Condition=" '$(SkipBuildingInstallers)' != 'true' And !$(Architecture.StartsWith('arm')) "
+          Condition=" '$(SkipBuildingInstallers)' != 'true' "
           DependsOnTargets="Prepare;
                             Layout;
-                            GeneratePkgs;
-                            GenerateDebs;
-                            GenerateMsis;
-                            GenerateRpms" />
+                            GenerateMSBuildExtentions" />
```
Comment (dsplaisted) [inline build/Package.targets]: Still misspelled here

File: build/package/MSBuildExtensions.targets [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+                      '$(MSBuildExtensionsLayoutDirectory)'
+                      '$(FullNugetVersion)'
+                      '$(SdkMSBuildExtensionsNuspecFile)'
+                      '$(SdkMSBuildExtensionsNupkgFile)'" />
+
+    <GenerateMSBuildExtensionsSWR MSBuildExtensionsLayoutDirectory="$(MSBuildExtensionsLayoutDirectory)"
+                                    OutputFile="$(SdkMSBuildExtensionsSwrFile)"/>
+  </Target>
+
+  <Target Name="GenerateMSBuildExtentions"
```
Comment (dsplaisted) [inline build/package/MSBuildExtensions.targets]: Still misspelled here

**Review by dsplaisted — APPROVED** [review]

**Pull request merged**